src/pages/cart/cart.tsx

```diff
@@ -1,5 +1,68 @@
+import { useQuery } from "@tanstack/react-query";
+import { Link } from "react-router-dom";
+import { CartItem } from "../../utils/types";
+import Item from "./Item";
+import { getCart } from "../../utils/routes";
+import { formatPrice } from "../../utils/format";
+
 function Cart() {
-  return <div>Cart</div>;
+  const query = useQuery(["cart"], async () => await getCart());
+
+  if (query.isLoading) {
+    return <div>Loading...</div>;
+  }
+  if (query.isError) {
+    return <div>Something went wrong</div>;
+  }
+
+  const cart = query.data.data;
```

Comment:
Could mark this with a `TODO` to replace with the fn we wrote in the other branch to calculate the total, just so you don't forget to circle back once you get that merged (we might have already done in in the wip branch, I forget).